test/functional-portable/datastoresrp/noncloud/resources/mongodb_test.go

```diff
@@ -114,3 +114,58 @@ func Test_MongoDB_Recipe(t *testing.T) {
 
 	test.Test(t)
 }
+
+func Test_MongoDB_Recipe_Existing(t *testing.T) {
```

Comment:
Where do we test that the namespace exists? Can we also use a better (more specific) naming for the test?